docs/client-sdk/javascript/tutorials/build-an-xmtp-hello-world-app.md

```diff
@@ -0,0 +1,578 @@
+
+The app includes only the code required to accomplish these steps and little else. This helps keep the codebase relatively bite-sized so that developers of all skill levels can ease into learning the basics of building a messaging app with XMTP.
+
+Completing this tutorial doesn't cost you any Ether. At this time, all messaging with XMTP is free. To learn more, see [Will XMTP charge messaging fees](https://c293c1b1.website-9re.pages.dev/docs/dev-concepts/faq/#will-xmtp-charge-messaging-fees)?
+   
+
+## Prerequisites
+Be sure to have these prerequisites in place before you start the tutorial:
+
+* Node (LTS version) and npm to install app requirements and dependencies
+
+* Access to Chrome, Firefox, or Brave. The "GM!" app is designed to work with these browsers.
+
+* A wallet app browser extension installed, such as MetaMask or Coinbase Wallet, and an Ethereum account. For example, you can install the MetaMask wallet app browser extension and use it to create an Ethereum account you can use to send and receive XMTP messages. To learn more, see [How to create a MetaMask Wallet](https://myterablock.medium.com/how-to-create-or-import-a-metamask-wallet-a551fc2f5a6b).
+    * Beyond the purposes of this "GM!" app tutorial, XMTP can be used with more than just Ethereum accounts and Metamask or Coinbase Wallet. To learn more, see Wallet apps and blockchains supported by XMTP.
+
+* A code editor of your choice that you can use to build the app
+
+## Step 1. Create a React + Typescript + Tailwind app
+
+To create the React app + Typescript, run:
+
+```
+npx create-react-app my-app --template typescript
+```
+
+Let's go into our new React app and install Tailwind.css
+
+```bash
+cd my-app
+npm install -D tailwindcss
+```
+
+Next we need to create the config files which can be done via the following command:
+
+```bash
+npx tailwindcss init -p
+```
+
+Next open up the created `tailwind.config.js` file and add the following to `content`:
+
+```js title="tailwind.config.js"
+content: [
+    './src/**/*.{js,jsx,ts,tsx}',
+],
+```
+
+Next we need to add the Tailwind directives to the `src/index.css` file, add the following to the top of the file:
+
+```css title="src/index.css"
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+```
+
+Finally, let's run our app and see it come to life
+```bash
+npm run start
+```
+
+The app opens at localhost:3000 in your default browser.
+
+## Step 2. Build Connect wallet button
+
+Your "GM!" app needs a **Connect wallet** button that enables you to connect a wallet app, such as MetaMask.
+
+This app uses [Web3Modal](https://web3modal.com/), [Ethers](https://docs.ethers.io/v5/single-page/), and [WAGMI](https://wagmi.sh/) to connect to commonly used wallet apps.
+
+:::important
+
+Before starting this task, be sure to have a wallet app browser extension installed, such as MetaMask or Coinbase Wallet, and an Ethereum account. To learn more, see [Prerequisites](#prerequisites).
+
+:::
+
+### Install Web3Modal.
```

Comment:
```suggestion
### Install Web3Modal
```

Did you mean to dismiss this edit? =) Headings don't need to end with a period.

Reply:
Probably did this by accident! I meant to "Commut suggestion" via github. My mistake!